app/Http/Resources/V4/MangaResource.php

```diff
@@ -194,6 +202,8 @@ public function toArray($request)
             'mal_id' => $this->mal_id,
             'url' => $this->url,
             'images' => $this->images,
+            'approved' => $this->approved ?? true,
```

Comment:
shouldn't we have three states for this?
- yes approved
- not approved
- unknown